packages/frontend/components/Footer/Footer.module.scss

```diff
@@ -39,35 +40,35 @@
   .links {
     display: flex;
     margin: 32px 0;
-  
+
     .column {
       display: flex;
       flex-direction: column;
       margin-right: 160px;
       font-size: 18px;
     }
-  
+
     p {
       font-weight: bold;
       margin: 1px;
     }
-  
+
     a {
       opacity: 0.7;
-	    margin: 1px;
-	    color: $white;
-	    text-decoration: none;
-      cursor: url('../public/asacoco.cur'), default;
```

Comment:
do we have a way to avoid writing relative url like this instead using something similar like `paths` in typescript ?

Reply:
I'm pretty sure there is a way, I'll look into it